src/components/ExportButton/exportUtils.js

```diff
@@ -95,11 +95,12 @@ export const getMapHd = (
   exportConfig,
 ) => {
   const targetSize = size || map.getSize();
-
   // We create a temporary map.
   const div = document.createElement('div');
   div.style.width = `${targetSize[0]}px`;
   div.style.height = `${targetSize[1]}px`;
+  div.style.margin = `0 0 0 -50000px`; // we move the map to the left to be ensure it is hidden during export
```

Comment:
Was it not hidden before ?

Reply:
for some reason it showed when using the external export button with the web component